From 884ec4c654f1d07a387fdc1dae5640606369f254 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Sat, 6 Feb 2021 10:56:36 -0800
Subject: [PATCH] make: Add compiler includes in cflags

Fixes
 In file included from ../git/ubi-utils/ubiformat.c:47:
| ../git/include/common.h:22:10: fatal error: 'stdbool.h' file not found
| #include <stdbool.h>
|          ^~~~~~~~~~~

Upstream-Status: Pending
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 Makefile.am | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Makefile.am b/Makefile.am
index 5a6e77c..f0003d5 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -9,6 +9,7 @@ endif
 if WITHOUT_LZO
 AM_CPPFLAGS += -DWITHOUT_LZO
 endif
+AM_CPPFLAGS += -I$(shell $(CC) -print-file-name=include)
 
 sbin_PROGRAMS =
 sbin_SCRIPTS =
-- 
2.30.0

